helm/cluster-vsphere/templates/helmreleases/cloud-provider-vsphere-helmrelease.yaml

```diff
@@ -28,7 +28,7 @@ spec:
   kubeConfig:
     secretRef:
       name: {{ include "resource.default.name" $ }}-kubeconfig
-  interval: {{ ((.Values.helmReleases).cpi).interval | default "10m" }}
+  interval: {{ (((.Values.internal).helmReleases).cpi).interval | default "10m" }}
```

Comment:
we don't use this but i tidied it up in case we wanted to add this to the schema later